x-pack/plugins/index_lifecycle_management/public/application/sections/edit_policy/form/validations.ts

```diff
@@ -149,3 +150,119 @@ export const createPolicyNameValidations = ({
     },
   ];
 };
+
+type MinAgePhase = 'warm' | 'cold' | 'frozen' | 'delete';
```

Comment:
Thanks for adding this type, @sebelga! Could we use it in other places where we have `'warm' | 'cold' | 'frozen' | 'delete'`?

Reply:
Good idea, I exported the type globally.